01_stratified_classification/nbconverted/lasso_range_all_optimizers.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### LASSO parameter range experiments, summary across all genes
```

Comment:
Maybe a quick word here about what liblinear and SGD (stochastic gradient descent, I'm assuming?) are and why you care about the differences?